completion/git-machete.completion.zsh

```diff
@@ -65,8 +65,18 @@ _git-machete() {
           _arguments '1:: :__git_machete_directions_go' && ret=0
           ;;
         (github)
-          _arguments '1:: :__git_machete_github_subcommands' && ret=0
-          ;;
+          case ${line[2]} in
```

Comment:
Check the zsh completion for a similar command! https://github.com/bobthecow/git-flow-completion/blob/master/git-flow-completion.zsh#L83

Reply:
This is also 3-level: `git flow hotfix start` etc.